keras_hub/src/models/gemma3/gemma3_presets.py

```diff
@@ -0,0 +1,13 @@
+"""Gemma model preset configurations."""
+
+# Metadata for loading pretrained model weights.
+backbone_presets = {
+    "gemma3_instruct_4b": {
```

Comment:
If we are just releasing the 1b first, let's just update this to be the 1b, and update any preset tests to use the 1b.